client/settings/general-settings-section/index.js

```diff
@@ -99,7 +100,7 @@ const GeneralSettingsSection = ( {
 							onCancel={ () => onChangeDisplayOrder( false ) }
 						/>
 					</AccountRefreshingOverlay>
-					{ isUpeEnabled && <SectionFooter /> }
+					{ isUpeEnabled && isPMCEnabled && <SectionFooter /> }
```

Comment:
```suggestion
					{ isUpeEnabled && ! isPMCEnabled && <SectionFooter /> }
```

Shouldn't we hide the section when `isPMCEnabled` is `True`?

Reply:
Ahh such stupid mistake 🙄 
Thanks Malith.